### ISSUE TITLE
The default configuration of the dock should use the "modern" style.

### ISSUE THREAD

**Issue by KyleCardoza** [AgoraDesktop/AgoraInstaller#18]
This involves setting the default "dockstyle" to the integer value 0.

**Comment by eukara** [comment]
Also on a side note, any defaults not specific to applications (any NSGlobalDomain stuff) goes right into `/etc/GNUstep/GlobalDefaults.plist`. That's specifically meant for distros

**Comment by KyleCardoza** [comment]
Makes sense -- can I set those with `sudo defaults write /etc/GNUstep/GlobalDefaults.plist NSSomeDefault "SomeStringValue"`?

**Comment by eukara** [comment]
No, you can neither save it directly into that file with the `defaults` tool, nor can you set application specific defaults as far as I can tell. It's basically `~/Library/Preferences/NSGlobalDomain.plist`, but standard for all users

**Comment by KyleCardoza** [comment]
Ah, so I'll have to make a new plist and copy it with `sudo`. Well, that can be done.
